docs/zope2book/AdvZPT.rst

```diff
@@ -1202,6 +1202,27 @@ caching for performance-critical applications.
 For more information on caching in the context of Zope, see the
 chapter entitled `Zope Services <ZopeServices.html>`_.
 
+Filesystem caching for Chameleon-based templates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Zope 4 introduced the `Chameleon HTML/XML template engine
+<https://chameleon.readthedocs.io/>`_ as new backend for Zope Page
+Templates. The Chameleon templating engine can compile templates and
+cache them on the file system for faster startup and execution.
+
+File system caching is activated by setting an environment variable
+named ``CHAMELEON_CACHE`` to the path of a folder on the filesystem
+where Chameleon can write its compiled template representation.
+
+Look for or add a section named ``environment`` in ``etc/wsgi.conf``
+and add a suitable filesystem path, for example::
+
+  <environment>

```

Comment:
Cool, in our projects we always put this variable directly into `os.environ`, because at least in Zope 2 `environment` was loaded to late to have any effect for the chameleon configuration.

Reply:
I just did some light testing and saw those compiled files show up when using the `<environment>` section, so I can tell it works.

Reply:
I also did that kind of testing to convince myself that it works now.